File: Week3/Day2/Solutions/Python/prog2.py
num1 = int(input("num1:"))
num2 = int(input("num2:"))

for i in range(num1,num2+1):
    fact = 1
    for j in range(1,i+1):
        fact = fact*j
    print("Factorial of",i,":",fact)
print()

File: Week3/Day2/Solutions/C/prog2.c
int printf(const char*,...);
int scanf(const char*,...);

void main() {
	int num1,num2,i,j,fact;
	scanf("%d %d",&num1,&num2);
	for(i=num1;i<=num2;i++){
		fact = 1;
		for(j=1;j<=i;j++){
			fact*=j;
		}
	printf("Factorial of %d is : %d\n",i,fact);
	}
}
